components/_global-navigation.scss

```diff
@@ -57,10 +59,52 @@ $nav-width: 1024px;
 		}
 	}
 
+	&__icon-user {
```

Comment:
Do you suggest making it a separate component? That makes sense—avatar is also used e.g. on discussions in posts.